### PR TITLE
Updating Expo/Firestore docs

### PR DESCRIPTION
**Pull request by masaok**
# Why

Expo supports Firestore, but the latest docs have not been updated.  After chatting with Brent and Evan in Slack, I decided to contribute to the docs.

# How

Built using Expo on Mac (iOS) and Windows (Android)

# Test Plan

Hmmm... no test plan yet.  Although it works in the Expo app that I'm working on.  If you have any suggestions, please let me know!

### CODE REVIEW

**Comment by EvanBacon** [comment]
thanks for this

File: docs/pages/versions/v32.0.0/guides/using-firebase.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Luckily, the Firebase JavaScript SDK starting from version 3.1+ has almost full
...
 
-> **Note:** This guide only covers Firebase Realtime Database, and not the other services under the larger Google Firebase umbrella. Firebase Cloud Storage is currently not supported, but we are [working on upstreaming a Blob implementation](https://github.com/facebook/react-native/issues/11103) to React Native that would make this possible. For more background on why other Firebase services are not supported, please read [Brent Vatne's response on Canny](https://expo.canny.io/feature-requests/p/full-native-firebase-integration)
+> **Note:** This guide mostly covers Firebase Realtime Database (and some Firestore as well), but not the other services under the larger Google Firebase umbrella. Firebase Cloud Storage is currently not supported, but we are [working on upstreaming a Blob implementation](https://github.com/facebook/react-native/issues/11103) to React Native that would make this possible. For more background on why other Firebase services are not supported, please read [Brent Vatne's response on Canny](https://expo.canny.io/feature-requests/p/full-native-firebase-integration)
```
Comment (EvanBacon) [inline docs/pages/versions/v32.0.0/guides/using-firebase.md]: could you update this, storage has been possible for some time now

File: docs/pages/versions/v32.0.0/guides/using-firebase.md [hunk 2]
```diff
@@ -166,3 +166,26 @@ function storeHighScore(user, score) {
...
+const firebaseConfig = { ... }  // apiKey, authDomain, etc. (see above)
+
+firebase.initializeApp(firebaseConfig);
+
+const dbh = firebase.firestore();
+
+dbh.collection("characters").doc("mario").set({
+  employment: "plumber",
+  outfitColor: "red",
+  specialAttack: "fireball"
```
Comment (EvanBacon) [inline docs/pages/versions/v32.0.0/guides/using-firebase.md]: nice

**Review by EvanBacon — CHANGES_REQUESTED** [review]
Just update the note about storage not being available.

File: docs/pages/versions/v32.0.0/guides/using-firebase.md (hunk 1 above)
Comment (masaok) [reply docs/pages/versions/v32.0.0/guides/using-firebase.md]: Sure, I will try, although I've never used it myself (I've only seriously been using Expo for a few weeks now).  Looks like this is the package: https://www.npmjs.com/package/expo-firebase-storage ... yes?

File: docs/pages/versions/v32.0.0/guides/using-firebase.md (hunk 2 above)
Comment (masaok) [reply docs/pages/versions/v32.0.0/guides/using-firebase.md]: thanks!

File: docs/pages/versions/v32.0.0/guides/using-firebase.md (hunk 1 above)
Comment (EvanBacon) [reply docs/pages/versions/v32.0.0/guides/using-firebase.md]: nope you can use web storage with Expo. That library is used for native firebase storage.

Comment (EvanBacon) [reply docs/pages/versions/v32.0.0/guides/using-firebase.md]: https://github.com/EvanBacon/firebase-instagram/blob/master/utils/uploadPhoto.js

Comment (masaok) [reply docs/pages/versions/v32.0.0/guides/using-firebase.md]: As I stated in my commit, I added text that I thought would be helpful to future "new people" (myself included) who are just starting to use Expo with Firebase (and might be having trouble navigating the docs and code).

**Comment by EvanBacon** [comment]
Looks good. Could you move the changes to this file: `docs/pages/versions/unversioned/guides/using-firebase.md` Sorry, I didn't notice this sooner.

**Comment by masaok** [comment]
> Looks good. Could you move the changes to this file: `docs/pages/versions/unversioned/guides/using-firebase.md` Sorry, I didn't notice this sooner.

Move or copy?  I'll copy first, then revert the v32 file if you confirm you prefer a "move".

**Comment by masaok** [comment]
> Just update the note about storage not being available.

**Comment by masaok** [comment]
@EvanBacon this PR is failing checks at some point in the `publish.sh` script.  I have no idea how the CI stuff works, do you?

image

**Comment by brentvatne** [comment]
the failure should be unrelated to docs, thanks!

**Pull request merged**